packages/@aws-cdk/aws-ec2/lib/bastion-host.ts

```diff
@@ -64,6 +65,14 @@ export interface BastionHostLinuxProps {
    * may be replaced on every deployment).
    */
   readonly machineImage?: IMachineImage;
+
+  /**
+   * Encryption for EBS volume
+   * If true, encrypted volume will be created with a default voulme size of 10 GiB.
+   *
+   * @default false
+   */
+  readonly ebsVolumeEncryption?: boolean;
```

Comment:
I think if we're going to go here, we need full EBS device configuration support, just like we have on `ec2.Instance`.

Reply:
Thanks for the feedback!

I have made the requested changes.